api/frontend.proto

```diff
@@ -12,12 +12,49 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-syntax = 'proto3';
+syntax = "proto3";
 package api;
 option go_package = "internal/future/pb";
 
-import 'api/messages.proto';
-import 'google/api/annotations.proto';
+import "api/messages.proto";
+import "google/api/annotations.proto";
+import "protoc-gen-swagger/options/annotations.proto";
+
+
+option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
+	info: {
+		title: "Frontend";
+		version: "1.0";
+		contact: {
+			name: "Open Match";
+			url: "https://open-match.dev";
+			email: "open-match-discuss@googlegroups.com";
+		}
+		license: {
+			name: "Apache 2.0 License";
+			url: "https://github.com/GoogleCloudPlatform/open-match/blob/master/LICENSE";
+		}
+	}
+	external_docs: {
+		url: "https://open-match.dev/site/docs/";
+		description: "Open Match Documentation";
+	}
+	schemes: HTTP;
+	schemes: HTTPS;
+	consumes: "application/json";
+	produces: "application/json";
+	responses: {
```

Comment:
Are we going to add the [security annotations](https://github.com/grpc-ecosystem/grpc-gateway/blob/master/examples/proto/examplepb/a_bit_of_everything.proto#L80) at here? I am not sure if users are still able to trigger api calls from swagger ui once the TLS support is enabled.

Reply:
Yes but not right now. Will add a TODO.